yb-voyager/cmd/common.go

```diff
@@ -1122,6 +1123,7 @@ func getMigrationComplexityForOracle(schemaDirectory string) (string, error) {
 
 // =====================================================================
```

Comment:
Looks like this file is getting too long. Shall we create a separate file in cmd for `assessment.go` or something like that?

Reply:
makes sense. was thinking about it.